dht/i_partitioner.cc

```diff
@@ -472,11 +472,9 @@ split_ranges_to_shards(const dht::token_range_vector& ranges, const schema& s) {
 namespace std {
 
```

Comment:
Please add an explanation in the changelog about why this is desirable safe. In general changelogs should explain whys, not just whats.

Reply:
done